src/cubehelix.jl

```diff
@@ -0,0 +1,45 @@
+
+# following
+#  https://www.mrao.cam.ac.uk/~dag/CUBEHELIX/
+# original source in f77
+#  https://www.mrao.cam.ac.uk/~dag/CUBEHELIX/cubhlx.f
+# 
+# published in 
+#  Green, D. A., 2011, `A colour scheme for the display of astronomical intensity images', Bulletin of the Astronomical Society of India, 39, 289.
+#    (2011BASI...39..289G at ADS.) 
+
+import Color: RGB
+
+# parameters: N number of entries in result colormap (btw: Array(RGB{Float64},N))
+#  rots: rotations -> added term
+#  start: startpoint in H -> 0.0 -> Red
+#  hue: saturation of hue -> 0.0 -> grayscale only, >1.0 -> clipping in some areas
+#  gamma: gamma factor 
+
+function cubehelix(N::Int,rots::Real,start::Real,hue::Real,gamma::Real)
```

Comment:
I agree with all of the above. The only limited valid range of input arguments seems to be hue and gamma.